### ISSUE TITLE
Initialising a PostCollection with an array

### ISSUE THREAD

**Issue by maikelRAOW** [offbeatwp/framework#46]
It is currently possible to create a PostCollection with an array of PostModels. However, this will cause issues if a plugin attempts to call the getQuery Method. Query is only set if a PostCollection is created with a WP_Query.

**Comment by ChrisRAoW** [comment]
@maikelRAOW I do not think it this is something that can or should be "solved".  What do you think?

**Comment by maikelRAOW** [comment]
@ChrisRAoW You could generate a query that selects all the IDS of the posts that are currently in the collection.
`$this->query = new WP_Query( 'post_type' => $type, ‘post__in' => $ids);`

**Comment by ChrisRAoW** [comment]
@maikelRAOW Mhh interesting.....could work indeed.
